components/admin_console/permission_schemes_settings/edit_post_time_limit_button/index.tsx

```diff
@@ -5,13 +5,17 @@ import {connect} from 'react-redux';
 
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
+import {GlobalState} from 'types/store';
+
+import {Constants} from 'utils/constants';
+
 import EditPostTimeLimitButton from './edit_post_time_limit_button';
 
-function mapStateToProps(state) {
+function mapStateToProps(state: GlobalState) {
```

Comment:
@M-ZubairAhmed Should we use the redux hooks to make remove the need for this file? The state is simple enough to make sense to avoid the whole index file.

Reply:
@sinansonmez From my side, you can leave the PR as it is. If you have curiosity of how the conversation went, you can take a look here: https://community.mattermost.com/core/pl/yb6s4i4mx388zrxuppcm1nypae

Reply:
Thanks @larkox for sharing the conversation